js/modules/authDOMHandler.js

```diff
@@ -0,0 +1,31 @@
+const deleteWarningText = (element) => {
+  const deleteTarget = element.parentElement.lastElementChild;
+  if (deleteTarget.className === "alert-danger") {
+    deleteTarget.remove();
+  }
+};
+
+const toggleWarningborder = (element) => {
+  if (element.parentElement.lastElementChild.className === "alert-danger") {
+    element.classList.toggle("warning-alert-border");
+  }
+};
+
+const createWarningText = (element, message) => {
+  if (element.parentElement.lastElementChild.className !== "alert-danger") {
+    const warningText = document.createElement("div");
+    warningText.textContent = message;
+    warningText.classList.add("alert-danger");
+    element.parentElement.appendChild(warningText);
+  } else {
+    element.parentElement.lastElementChild.textContent = message;
+  }
+  toggleWarningborder(element);
+};
+
+const specifyWarningPosition = (element, text) => {
+  toggleWarningborder(element);
+  createWarningText(element, text);
+};
```

Comment:
- 사용하시는 곳들을 보니, 이런 제안이 떠오릅니다. 
  - 함수이름에 "경고 위치"라는 표현이 있는데, 구현내용에는 position이나 좌표에 대한 내용은 없어서 직관적으로 이해하기힘들었습니다. 사실 위치position에 대한 내용은 아니었던거같아요
  - 매개변수가 의미하는 바를 보니, 매개변수 네이밍을 더 정확하게 할 수 있을거같아요.

```suggestion
const triggerInputValidationError = (input, message) => {
  toggleWarningborder(input);
  createWarningText(input, message);
};
```

Reply:
좀 더 명확하고 직관적인 이름. 참고하겠습니다. 이름 짓는게 정말 쉽지 않네요 ㅜ